client/src/main/java/io/apicurio/registry/client/RegistryClient.java

```diff
@@ -31,6 +31,7 @@
  */
 public class RegistryClient {
     private static final Logger log = Logger.getLogger(RegistryClient.class.getName());
+    private static String token;
```

Comment:
We should manage the auth logic for the client. It should take care of authentication and refreshing implementation so the users should not worry about auth stuff.

Reply:
@carlesarnal: Made changes, please review again